### ISSUE TITLE
2.0.15 broke saving submissions (fine in 2.0.14)

### ISSUE THREAD

**Issue by JeanLucEsser** [verbb/formie#1160]
### Describe the bug

When going into a submission via the CP and trying saving it I have the following error:

```
Unknown Method – [yii\base\UnknownMethodException](https://www.yiiframework.com/doc-2.0/yii-base-unknownmethodexception.html)
Calling unknown method: verbb\formie\models\FieldLayout::getFields()
```

This is new going from 2.0.14 to 2.0.15.

I didn't test it further yet (field types) but can if needed.
Should you need more logs, ask me.

### Steps to reproduce

1. Go to a submission in the CP
2. Try saving it

### Form settings

- Multi-page form: Yes
- Submission Method: Page Reload
- Client-side Validation: No
- Custom Form Templates: No

### Craft CMS version

4.3.1

### Plugin version

2.0.15

### Multi-site?

No

**Comment by benoitblancquaert** [comment]
I have the same issue on version `2.0.15`.

```
Unknown Method – [yii\base\UnknownMethodException](https://www.yiiframework.com/doc-2.0/yii-base-unknownmethodexception.html)
Calling unknown method: verbb\formie\models\FieldLayout::getFields()
```

Tested on some forms with different settings.

**Comment by MaartenWaegeman** [comment]
I guess https://github.com/verbb/formie/commit/3547b0da84409e3aa55697fbd8348d97232e12eb#diff-a362fa4873f7f958b34182f90ab1949610476f9bf343b48f8b8646b1c0cf8e93R761 breaks it, as getFields isn't defined on a FieldLayout.

**Comment by engram-design** [comment]
Sorry about that, fixed in [2.0.16](https://github.com/verbb/formie/releases/tag/2.0.16)
